gigboard-client/src/pages/Register/Register.jsx

```diff
@@ -1,27 +1,63 @@
+import { useState } from 'react';
+
+import { useHistory } from 'react-router-dom';
 import { Button, Form } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css'
 
+import Auth from '../../Models/Auth'
+
+
 const Register = () => {
+
+    const history = useHistory()
+    
+    const [ email, setEmail ] = useState('')
+    const [ password, setPasword ] = useState('')
+    const [ username, setUsername ] = useState('')
+
+    const [ message, setMessage ] = useState( null );
+    
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+    
+        try {
+          const user = await Auth.register({ email, username, password });
```

Comment:
So to register a user we're going to be needing more information to create a user in the backend. We'll need an email, username, firstName, lastName, password and password2. Again this might need to be discussed with the rest of the group but i think two passwords to verify they spelled their first password correctly would be good